client/src/components/Feed/PostCard.js

```diff
@@ -95,30 +97,38 @@ const PostCard = styled(Card)`
         }
 
         .author,
-        .location-status {
+        .sub-header {
           line-height: 2rem;
+          font-size: ${xsmall};
         }
 
         .author {
           font-size: ${medium};
+          cursor: pointer;
         }
 
         .location-status {
           position: relative;
           padding-left: 1.4rem;
           font-size: ${xsmall};
           color: #888;
-
+          cursor: default;
           img {
             position: absolute;
-            top: 0.7rem;
+            top: 0.5rem;
             left: 0;
             width: 0.6rem;
             height: 0.6rem;
             margin-right: 0;
           }
         }
       }
+      .timestamp {
+        padding-left: 1rem;
+        font-size: ${xsmall};
+        color: #888;
+        cursor: pointer;
```

Comment:
```suggestion
```

Since we removed the link from this part of the header we should not have a pointer here.

![image](https://user-images.githubusercontent.com/10546720/99189181-0d4c1180-272e-11eb-9346-48ff1bab0bbd.png)